hazelcast/src/test/java/com/hazelcast/internal/partition/GracefulShutdownTest.java

```diff
@@ -328,7 +340,7 @@ public void run() {
         // spin until node starts to shut down
         Node shuttingDownNode = getNode(shuttingDownInstance);
         while (shuttingDownNode.isRunning()) {
-            ;
+            sleepMillis(50);
```

Comment:
Sleeping 50ms can be too much for this test. I suggest to use at most 1ms, maybe even smaller.

Reply:
I replaced it with `Thread.yield()` to just give the scheduled a hint. WDYT?